docs/user-guide/release-notes/2.12.x.rst

```diff
@@ -2,6 +2,20 @@
 Pulp 2.12 Release Notes
 =======================
 
+
+EL6 Support Changes
+===================
+
+The 2.12 releases no longer support deploying Pulp itself on EL6. This does not change the use
+cases of running ``pulp-agent`` on EL6 or whether Pulp can sync and publish EL6 repositories,
+which will still be supported.
```

Comment:
This seems a bit muddled. What about:

The 2.12 releases no longer support deploying Pulp on EL6. However, running ``pulp-agent`` on EL6 and syncing/publishing EL6 repositories will still be supported.